mkall/MKReporterInternal.h

```diff
@@ -0,0 +1,25 @@
+// Part of Measurement Kit <https://measurement-kit.github.io/>.
+// Measurement Kit is free software under the BSD license. See AUTHORS
+// and LICENSE for more information on the copying conditions.
+#ifndef MKReporterInternal_h
+#define MKReporterInternal_h
+
+// Internal file containing interfaces used for testing.
+
+#import "MKReporter.h"
+#import "vendor/mkcollector.hpp"
+
+struct MKReporterStats {
+#define XX(name_) unsigned name_;
+  MKCOLLECTOR_REPORTER_STATS_ENUM(XX)
+#undef XX
+};
```

Comment:
Because this is now internal, we can probably use the mkcollector.hpp struct directly